tests/product/base_product_case.py

```diff
@@ -187,7 +187,7 @@ def dump_and_cp_topology(self, topology, cluster=None):
             cluster = self.cluster
         cluster.write_content_to_host(
             json.dumps(topology),
-            '/etc/opt/prestoadmin/config.json',
+            '~/.prestoadmin/config.json',
```

Comment:
Yea you're right, we should haven working commits. Before pushing can you squash product changes with all test changes?